cloudgarden-pwa/src/components/main/LoginForm.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import OktaAuth from "@okta/okta-auth-js";
+import { useOktaAuth } from "@okta/okta-react";
+
+interface LoginFormProps {
+  baseUrl: String;
+}
+
+const LoginForm = ({ baseUrl }: LoginFormProps) => {
+  const { authService } = useOktaAuth();
+  const [sessionToken, setSessionToken] = useState();
+  const [username, setUsername] = useState();
+  const [password, setPassword] = useState();
+
+  const handleSubmit = (e: any) => {
```

Comment:
This is where the problem is, but I do not know why since I pretty much copied this exactly from here: https://developer.okta.com/code/react/okta_react/